src/itemrenderer.cpp

```diff
@@ -130,6 +130,22 @@ void render_html(
 	}
 }
 
+void render_plaintext(
+	const std::string& source,
+	std::vector<std::pair<LineType, std::string>>& lines)
+{
+	std::string::size_type pos = 0;
+	while (pos < source.size()) {
+		const auto end_of_line = source.find_first_of("\r\n", pos);
```

Comment:
I guess it makes sense but I see a few question marks due to `\r` characters in articles from youtube feeds.

For example, the `Why Bayes rule is nicer with odds` entry from https://www.youtube.com/feeds/videos.xml?channel_id=UCYO_jab_esuFRV4b17AJtAw, which shows multiple question marks in the lower half of the description:
![image](https://user-images.githubusercontent.com/4629607/114260834-741ce080-99d7-11eb-95e0-1287fe0fc190.png)

Reply:
Looks like the description includes the `\r` characters as entity references (`&#13;`) so that explains why libxml2 does not normalize them:
![image](https://user-images.githubusercontent.com/4629607/114263274-d6301280-99e4-11eb-8947-4ef4072f8f6d.png)

Reply:
Got it; thanks for the investigation! Let's leave this as-is, then.

Reply:
By "as-is" I mean "roll back to using `\r\n` (as it was when this discussion was started)". Sorry for the confusion, it's just now that I'm re-reviewing the updated PR that I notice you've changed it to `\n` already.